releasenotes/README.md

```diff
@@ -26,27 +26,24 @@ Additional documentation:
 
 ------------
 
-* CHANGE: We removed the `--target-dev-hub` flag from these commands; the flag has been deprecated and hidden for over two years because it has no effect:
+* CHANGE: We made a bunch of improvements to the `data import tree` and `data export tree` commands. But because these improvements introduce breaking changes, we're going to first beta the new functionality, and then make it generally available when we're sure everything is in tip-top shape. Here's the plan:
 
-    * `org create user`
-    * `org display user`
-    * `org list users`
- 
-    (plugin-users PR [#864](https://github.com/salesforcecli/plugin-user/pull/864))
+    * This Salesforce CLI release contains two new commands that contain the improvements: `data import beta tree` and `data export beta tree`. The existing `data import tree` and `data export tree` commands continue to work as before.
+    * Read the [pinned issue](https://github.com/forcedotcom/cli/issues), which details the improvements in these beta commands, as well as the breaking changes. We also provide a tentative timeline.
+    * Test these beta commands and let us know if you find any issues. Also let us know if you like the improvements!
+    * When the beta period is over, we'll move the functionality that we added to `data import|export beta tree` to the "official" `data import|export tree` commands. We then move the functionality in the _old_ `data import|export tree` commands to new commands called `data import|export legacy tree`. See the [pinned issue](https://github.com/forcedotcom/cli/issues) for more information. 
 
-* FIX: We updated our Github Actions to be more friendly to external developers. In particular, we modified our [external plugin template](https://github.com/salesforcecli/plugin-template-sf-external) so that `dev generate plugin` generates updated sample GitHub Actions workflow files. We also added Personal Access Token instructions to the [README](https://github.com/salesforcecli/github-workflows) in our GitHub Actions repo.
-
-* FIX: Salesforce DX projects now support these [metadata types](https://github.com/forcedotcom/source-deploy-retrieve/blob/main/src/registry/metadataRegistry.json):
+    Enjoy! (GitHub issues [#2663](https://github.com/forcedotcom/cli/issues/2663) and [#248](https://github.com/forcedotcom/cli/issues/248). GitHub discussion [#2359](https://github.com/forcedotcom/cli/discussions/2359). plugin-data PR [#821](https://github.com/salesforcecli/plugin-data/pull/821) and [#810](https://github.com/salesforcecli/plugin-data/pull/810))
 
-    * EnablementMeasureDefinition
+* FIX: The `sf which` command, when passed an alias of a deprecated command, now outputs the name of the new command rather than the old deprecated one. (oclif GitHub issue [#515](https://github.com/oclif/plugin-which/issues/515), plugin-which PR [#545](https://github.com/oclif/plugin-which/pull/545))
```

Comment:
This fix isn't specific to deprecated commands. It'd be more accurate to say,

>FIX: The `sf which` command, when passed an alias of a command, now outputs an `aliasOf` property that shows the command that the provided command is an alias of.

Reply:
Thanks!  Because there are so many "commands" in that sentence, I edited it a bit; is this okay?  

> When you pass a command alias to `sf which`, it now outputs an `aliasOf` property that shows the command that the passed-in command is an alias of.